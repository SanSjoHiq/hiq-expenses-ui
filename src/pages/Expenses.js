import React from 'react'
import ExpensesForm from '../components/ExpensesForm'

export const Expenses = () => {

    return(
        <>
            <ExpensesForm />
        </>
    )
}
