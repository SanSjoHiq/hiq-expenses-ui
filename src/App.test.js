import React from 'react'
import {render, cleanup} from '@testing-library/react'
import { Expenses } from './pages/Expenses'

 afterEach(cleanup)
 
 test('should generate a snapshot for app component', () => {
    const { asFragment } = render(<Expenses />)
    
    expect(asFragment(<Expenses />)).toMatchSnapshot()
   });