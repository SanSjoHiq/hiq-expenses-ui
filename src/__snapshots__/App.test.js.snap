// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate a snapshot for app component 1`] = `
<DocumentFragment>
  <div
    class="MuiFormControl-root makeStyles-form-3 css-1nrlq1o-MuiFormControl-root"
  >
    <label
      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
      data-shrink="false"
      id="select-label"
    >
      Välj typ av utlägg
    </label>
    <div
      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiSelect-root makeStyles-select-4 css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
      select="true"
    >
      <div
        aria-expanded="false"
        aria-haspopup="listbox"
        aria-labelledby="select-label outlined-select-currency"
        class="MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
        id="outlined-select-currency"
        role="button"
        tabindex="0"
      >
        <span
          class="notranslate"
        >
          ​
        </span>
      </div>
      <input
        aria-hidden="true"
        class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
        data-testid="select-option"
        id="select-option"
        tabindex="-1"
        value=""
      />
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
        data-testid="ArrowDropDownIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
      <fieldset
        aria-hidden="true"
        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="css-dhh0px"
        >
          <span>
            Välj typ av utlägg
          </span>
        </legend>
      </fieldset>
    </div>
    <div
      class="MuiFormControl-root MuiTextField-root makeStyles-input-2 css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
    >
      <label
        class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-required css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink="false"
        for="outlined-required"
        id="outlined-required-label"
      >
        Summa
        <span
          aria-hidden="true"
          class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
      >
        <input
          aria-invalid="false"
          class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
          id="outlined-required"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-dhh0px"
          >
            <span>
              Summa *
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <button
      class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root makeStyles-button-1 css-sghohy-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      Lägg till
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
